testsuite/MDAnalysisTests/analysis/test_base.py

```diff
@@ -226,28 +252,42 @@ def test_frame_slice(u_xtc, run_kwargs, frames):
     ({'start': 4, 'step': 2, 'frames': [4, 6, 8]}),
     ({'start': 0, 'stop': 0, 'step': 0, 'frames': [4, 6, 8]}),
 ])
-def test_frame_fail(u, run_kwargs):
+def test_frame_fail(u, run_kwargs, client_FrameAnalysis):
     an = FrameAnalysis(u.trajectory)
     msg = 'start/stop/step cannot be combined with frames'
     with pytest.raises(ValueError, match=msg):
-        an.run(**run_kwargs)
+        an.run(**client_FrameAnalysis, **run_kwargs)
+
+def test_parallelizable_transformations():
+    from MDAnalysis.transformations import NoJump
+    u = mda.Universe(XTC)
+    u.trajectory.add_transformations(NoJump(parallelizable=True))
```

Comment:
I don't understand what the test is doing.

It does not really make sense to set `parallelizable=True` for `NoJump` because it is inherently _not parallelizable_.

And even if `parallelizable=True` then why does the parallel execution below fail with ValueError?

Reply:
>It does not really make sense to set parallelizable=True for NoJump because it is inherently not parallelizable.
I just wanted to have any transformation that would allow me to set this attribute. Should I pick another one?

>And even if parallelizable=True then why does the parallel execution below fail with ValueError?
You're right, it should be that a) `NoJump(parallelizable=False)` b) serial works and c) serial fails. I changed the code of `AnalysisBase` and the test.